traitsui/qt4/video_editor.py

```diff
@@ -233,10 +251,19 @@ def init(self, parent):
         )
         self.control.setBackgroundRole(QPalette.ColorRole.Window)
 
-        self.media_player = QMediaPlayer(None, QMediaPlayer.VideoSurface)
+        if is_qt5:
+            self.media_player = QMediaPlayer(None, QMediaPlayer.VideoSurface)
+        else:
+            self.media_player = QMediaPlayer()
         self._set_video_url()
         self.media_player.setVideoOutput(self.control)
-        self.media_player.setMuted(self.muted)
+        if not is_qt5:
```

Comment:
I would recommend `if is_qt5:` just to keep things consistent with the other conditional paths